tasks/bake.js

```diff
@@ -384,6 +384,25 @@ module.exports = function( grunt ) {
 			return null;
 		}
 
+		// Handle _bake attributes in inline arguments
+
+		function validateExtraBake( inlineValues ) {
```

Comment:
Was thinking about this as well and then run into this naming ambiguity question. Let's rename it here to have consistant naming in the form `validate<nameOfAttribute>`, as you said.